src/plugins/wb-data-json/conditional-en.hbs

```diff
@@ -0,0 +1,1488 @@
+			<li><code>rdfs:Literal</code> for any value that is literal. A literal value is considered to be defined and not an object.</li>
+			<li><code>xsd:anyURI</code> when the value is a string which start with non-spaced letter and digit immediately followed by colon ":" like <code>https://example.com</code></li>
+			<li><code>xsd:string</code> when the value is a string and don't look to be an URI.</li>
+			<li><code>xsd:boolean</code> when the value is a boolean with the value set to <code>true</code> or <code>false</code></li>
+			<li><code>xsd:double</code> when the value is a JSON number</li>
+			<li>The value set via the <code>@type</code> sub property</li>
+			<li><code>undefined</code> when the value is undefined</li>
+		</ul>
+	</dd>
+
+	<dt><code>fn:getValue</code></dt>
+	<dd>Return value as the testable value.</dd>
+</dl>
+
+<h3>Example of each test function</h3>
+
+<div class="row">
+	<div class="col-md-6">
+		<p>Demo of each test function.</p>
+		<ul data-wb-json='{
+			"url": "demo/conditional-2.json",
+			"streamline": true,
+			"mapping": [
+				{
+					"template": "[data-isarray]",
+					"test": "fn:isArray",
+					"assess": "/alpha",
+					"mapping": [
+						{ "selector": "span", "value": "/alpha/0" }
+					]
+				},
+				{
+					"template": "[data-isliteral]",
+					"test": "fn:isLiteral",
+					"assess": "/beta",
+					"mapping": [
+						{ "selector": "span", "value": "/beta" }
+					]
+				},
+				{
+					"template": "[data-isliteral]",
+					"test": "fn:isLiteral",
+					"assess": "/gamma",
+					"mapping": [
+						{ "selector": "span", "value": "/gamma" }
+					]
+				},
+				{
+					"template": "[data-gettype]",
+					"test": "fn:getType",
+					"assess": "/delta",
+					"expect": "rdfs:Resource",
+					"mapping": [
+						{ "selector": "span", "value": "/delta" }
+					]
+				},
+				{
+					"template": "[data-gettype]",
+					"test": "fn:getType",
+					"assess": "/epsilon",
+					"expect": "string",
+					"mapping": [
+						{ "selector": "span", "value": "/epsilon" }
+					]
+				},
+				{
+					"template": "[data-gettype]",
+					"test": "fn:getType",
+					"assess": "/zeta",
+					"expect": "boolean",
+					"mapping": [
+						{ "selector": "span", "value": "/text/zeta" }
+					]
+				},
+				{
+					"template": "[data-guesttype]",
+					"test": "fn:guestType",
+					"assess": "/alpha",
+					"expect": "rdfs:Container",
+					"mapping": [
+						{ "selector": "span", "value": "/alpha/0" }
+					]
+				},
+				{
+					"template": "[data-guesttype]",
+					"test": "fn:guestType",
+					"assess": "/beta",
+					"expect": "rdfs:Literal",
+					"mapping": [
+						{ "selector": "span", "value": "/beta" }
+					]
+				},
+				{
+					"template": "[data-guesttype]",
+					"test": "fn:guestType",
+					"assess": "/gamma",
+					"expect": "rdfs:Literal",
+					"mapping": [
+						{ "selector": "span", "value": "/gamma" }
+					]
+				},
+				{
+					"template": "[data-guesttype]",
```

Comment:
```suggestion
					"template": "[data-guesstype]",
```

Reply:
guestType sts all over this file.  I will leave it to you to do a find and replace in the code :)